src/style.css

```diff
@@ -0,0 +1,73 @@
+@font-face {
+  font-family: Avenir;
+  src: url(Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+.search__blocks {
+  width: 100%;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  outline: none;
+
+  color: #3d4e61;
+  font-family: Avenir;
+}
+
+.search__blocks:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search__blocks:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+
+  font-weight: 700;
+}
+
+.search__big {
+  height: 70px;
+  padding-left: 62px;
+
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search__big-wrapper {
+  position: relative;
+  margin: 20px 0;
+}
+
+.search__big-wrapper::before {
+  content: "";
+  position: absolute;
+  background-image: url(images/Search.svg);
```

Comment:
These styles are repeated in small, so i think you need to make default class .search and specify common styles for it and apply it for big search bar and then you can add "search search--small" for small search to specify specific styles for it